editor/utils/shortcuts.ts

```diff
@@ -47,3 +52,16 @@ export const useKeyboardShortcut = (
     }
   }, [handleKeyPress]);
 };
+
+// Handles scrolling down to element on key press
+export const scrollToElementOnKeyPress = (
+  event: KeyboardEvent,
+  shortcutKeyLookup: ShortcutKeyLookup[]
+) => {
+  const eventKey = event.key.toUpperCase();
```

Comment:
i think am starting to confuse myself here, but do we need these calls `toUpperCase` throughout the code? or is defining the keyboard shortcut as capital "A" enough? if they event keys are case sensitive, then `shift` is implied through our use of capitalization.

it make sense to me that we want to logically think about a keyboard shortcut as `shift` + `a` instead of `A`, but the casing operations in the code are throwing me off. am i just overthinking it? 🤔

Reply:
The reason i added it was bc if users are in Caps mode on their keyboard and then they try to shift + a its not going to work, so thats why im uppercasing their input so it will always work whether or not they are in caps

Reply:
ah, of course, i follow you now. you are ensuring that the keyboard shortcuts are actually case insensitive. it's `shift` + `a` or `shift` + `A`. But the element IDs _are_ case sensitive: they have to be upper case. 👍